disaster-recovery.md

```diff
@@ -0,0 +1,169 @@
+3. After removing the compromised key, the new key for the validator must be added to the `MultisigControl` contract by calling the `add_signer` function. The signature bundle is generated in the same way as for (2).
+4. The Vega CLI can be used to generate signatures, please execute `vega bridge erc20 -h` for further instructions.
+
+
+### Scenario 1.2: Vega hot key compromised
+
+In the event that the Vega hot key is compromised, it can be replaced using the steps below.
+
+1. Terminate the affected node.
+2. Use the Vega master key to generate a new hot key (refer to [generating keys](https://github.com/vegaprotocol/networks#generating-vega-keys)).
+3. Submit a transaction to the Vega network, signed with your master key, authorising the new hot key.
+4. Rejoin the network when the rest of the validators choose to restore the chain from a [checkpoint file](https://github.com/vegaprotocol/networks#restore-from-checkpoint).
+
+
+### Scenario 1.3: Vega master key compromised
+
+In the event that the Vega master key is compromised, one can assume that the validator's infrastructure has been rather deeply penetrated. Thus, a compromise of the Vega master key can be seen to imply that all other keys are compromised as well (with the exception of when a HSM has been used; however, as outlined above, even if the key is not known to the attacker it should still be treated as compromised).
+
+1. Since we assume in this case that the validator's ethereum key is also compromised - follow steps from [Scenario 1.1: Ethereum key compromised](https://github.com/vegaprotocol/networks/blob/disaster-recovery-docs/disaster-recovery.md#scenario-11-ethereum-key-compromised)
+2. Since we assume in this case the validator's your hot key is also compromised - follow steps 1 - 3 from [Scenario 1.2: Vega hot key compromised](https://github.com/vegaprotocol/networks/blob/disaster-recovery-docs/disaster-recovery.md#scenario-12-vega-hot-key-compromised)
+3. Inform the other validators via the appropriate channels to immediately remove the compromised node from the network:
+   1. Stop the network
+   2. Update the genesis config file to remove the compromised node
+   3. Restore from checkpoint using the instructions [here](https://github.com/vegaprotocol/networks#restore-from-checkpoint)
+4. Use newly generated ethereum key to associate ERC20 Vega tokens from the compromised node to the new master key. 
+5. Inform all token holders that delegated to the compromised node to move their delegation. 
+
+
+### Scenario 1.4: Tendermint key compromised
+
+In the event a tendermint key gets compromised, a third party cannot double sign messages in that validator's name. This is not highly critical if it happens to only one validator, but that validator should replace its key as soon as reasonable to not allow this situation to get worse through multiple compromises.
```

Comment:
I suspect this should be "a third party *can* sign messages" @Vegaklaus can you please confirm?

Reply:
Yep, that's a type, it should be 'can'

Reply:
👍